### ISSUE TITLE
Database pathname

### ISSUE THREAD

**Issue by danielptm** [JoakimLandstrom/Projektuppgift_webdev#1]
Jag tror att det skulle vara jätte bra om vi byter derby driver pathen så att det är lokal till projektet. Då kan vi ladda ner projektet och kör det på vår egen dator utan att ändra den varje gång.

**Comment by JoakimLandstrom** [comment]
Yes låter som en bra idé! Vi får fixa det imorgon. Tycker du vi ska lägga in hela databasen i repot också?

**Comment by danielptm** [comment]
Det kanske spelar ingen roll men jag tycker att det är bra om vi kan göra det och det inte orsaker problem.

//Daniel
073-108 98 54
